mapdata/WarcraftLegacies/AbilityData/Skin/1244672083.json

```diff
@@ -25,7 +25,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Increases the movement speed of units by 15% in 5000 radius."
+      "Value": "Increases the movement speed of units by \u003CS00J,DataA1,%\u003E% in 5000 radius."
```

Comment:
(Nitpick, optional) you could use data field referencing for the 5000 here too. It'd be something like <u003,Area1>.